helloworld-mutual-ssl-secured/configure-ssl.cli

```diff
@@ -29,10 +29,21 @@ batch
 # Add an application-security-domain in the undertow subsystem to map the client_cert_domain from the quickstart app to the http-authentication-factory
 /subsystem=undertow/application-security-domain=client_cert_domain:add(http-authentication-factory=quickstart-http-authentication)
 
+#generate the key pair that the server would use for its keystore
+/subsystem=elytron/key-store=applicationKS:generate-key-pair(alias=server, algorithm=RSA, key-size=2048, validity=365, credential-reference={clear-text=password}, distinguished-name="cn=localhost")
+
+#add the keystore into the default key-manager
+/subsystem=elytron/key-manager=applicationKM:write-attribute(name=key-store, value=applicationKS)
```

Comment:
Why is this needed? The default applicationKM already references applicationKS.

Reply:
removed